doc/sphinx_source/modules/internals.rst

```diff
@@ -125,23 +146,65 @@ Triggering any Bind
+| BIND_WANTRET      | With stacked binds, if the called Tcl proc called returns a '1', halt processing any further binds triggered by the action    |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| BIND_STACKRET     | Used with BIND_WANTRET; allow stacked binds to continue despite receiving a '1'                                               |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| BIND_ALTER_ARGS   | Replaces arguments (which ones?) with the result returned from the called Tcl proc                                            |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+
+Bind Return Values
+^^^^^^^^^^^^^^^^^^
+The value returned by the bind is often matched against a desired value to return a '1' (often used with BIND_WANTRET and BIND_STACKRET) to the calling function.
+
++----------------+--------------------------------------------------------------------------------------------------------------+
+| **Value**      | **Description**                                                                                              |
++----------------+--------------------------------------------------------------------------------------------------------------+
+| BIND_NOMATCH   | The bind was not triggered due to not meeting the criteria set for the bind                                  |
++----------------+--------------------------------------------------------------------------------------------------------------+
+| BIND_AMBIGUOUS | The triggering action matched multiple non-stackable binds                                                   |
++----------------+--------------------------------------------------------------------------------------------------------------+
+| BIND_MATCHED   | The bind criteria was met, but the Tcl proc it tried to call could not be found                              |
++----------------+--------------------------------------------------------------------------------------------------------------+
+| BIND_EXECUTED  | The bind criteria was met and the Tcl proc was called                                                        |
++----------------+--------------------------------------------------------------------------------------------------------------+
+| BIND_EXEC_LOG  | The bind criteria was met, the Tcl proc was called, and Eggdrop logged the bind being called                 |
++----------------+--------------------------------------------------------------------------------------------------------------+
+| BIND_QUIT      | The bind was triggered in conjunction with the target leaving the partyline or filesys area (?)              |
```

Comment:
.quit is bound to a special value CMD_LEAVE that doesn't really trigger a bind but sets the Tcl result to the magic string "break" which is then used to detect this and signal the intention to quit the partyline/filesys by making the bind call return BIND_QUIT

Reply:
Can you write this one up

Reply:
> Sentinel value to signal `.quit` was triggered to leave the current partyline/filesys area (virtual bind to `CMD_LEAVE`)